packages/typespec-ts/test/modularIntegration/azureCore.spec.ts

```diff
@@ -159,4 +159,58 @@ describe("BasicClient Classical Client", () => {
     assert.strictEqual(items.length, 1);
     assert.strictEqual(items[0]?.name, "Madge");
   });
+
+  it("should export a user", async () => {
+    try {
+      const user = await client.exportOperation(1, "json");
+      console.log(user);
+      assert.strictEqual(user?.id, 1);
+      assert.strictEqual(user?.name, "Madge");
+      assert.strictEqual(user?.etag, "11bdc430-65e8-45ad-81d9-8ffa60d55b59");
+    } catch (err) {
+      assert.fail(err as string);
+    }
+  });
+
+  it.only("should create or replace a user", async () => {
+    try {
+      const user = await client.createOrReplace(1, {
+        name: "Madge",
+        id: 1,
+        etag: "11bdc430-65e8-45ad-81d9-8ffa60d55b59"
+      }, {
+        contentType: "application/json"
+      } as any);
```

Comment:
once core is fixed, we can pass the content type like this way or in the requestOptions.headers['content-type']